app/src/main/java/org/mozilla/fenix/compose/SectionHeader.kt

```diff
@@ -41,8 +41,39 @@ fun SectionHeader(
     )
 }
 
+/**
+ * Default layout for the header of a screen section.
+ *
+ * @param text [String] to be styled as header and displayed.
+ * @param modifier [Modifier] to be applied to the [Text].
+ */
+@Composable
+fun HomeSectionHeader(
+    text: String,
+    modifier: Modifier = Modifier
+) {
+    Text(
+        modifier = modifier,
+        text = text,
+        style = TextStyle(
```

Comment:
We should long term probably make a file of JUST text styles

Reply:
should I file something for that?

Reply:
Sure, I think that makes sense. We might also want to have a Compose Consortium meeting or something haha to discuss how to scale all of this now that it's really being implemented.

Reply:
@eliserichards @MozillaNoah I've created https://github.com/mozilla-mobile/fenix/issues/21192 and started working on a composables theme but the specs are not finished and abandoned the work for now.